### ISSUE TITLE
Formatter gets stuck on function imports from a module

### ISSUE THREAD

**Issue by PabloSichert** [nicklockwood/SwiftFormat#298]
Hi, first of all - thanks for providing this awesome formatting tool!

Running `swiftformat` seems to get stuck on a file containing:

```swift
import func foo.function
```

```bash
$ swiftformat --version
swiftformat, version 0.35.2
```

**Comment by nicklockwood** [comment]
@pablosichert Thanks for letting me know - I’ll investigate.

**Comment by nicklockwood** [comment]
@PabloSichert I've pushed a fix to the develop branch. If you'd prefer not to use a prerelease version, you can work around the problem in the current release by running swiftformat with `--disable redundantSelf`

**Comment by nicklockwood** [comment]
@PabloSichert this is fixed in version 0.35.4

**Comment by PabloSichert** [comment]
Awesome, confirming that the files in question format without problems now.

Thanks for the ⚡️-fast fix!
